src/test/setup.js

```diff
@@ -24,6 +24,7 @@ afterEach(() => {
   //
   // See https://github.com/facebook/jest/issues/7654
   jest.resetAllMocks();
+  jest.restoreAllMocks();
```

Comment:
Without "restore" I would still have the jest mocks assigned to the spied functions. I don't know why we didn't have this before...

Note: I tried to use restore without reset and got other problems. So I'm using both and this seems fine.